Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp

```diff
@@ -16,6 +16,15 @@
 
 using namespace Arm64Gen;
 
+#define CARRY_IF_NEEDED(inst_without_carry, inst_with_carry, ...)                                  \
```

Comment:
Is this not able to be made into a templated member function of JitArm64?

Reply:
It would be rather annoying, because most of the functions we want to pass as the first two arguments are overloaded, and I'm not of any way that a templated function would be able to automatically determine the correct overload without the caller having to explicitly specify the intended overload.